### ISSUE TITLE
compileOnSave setting has no effect when it's not declared in primary tsconfig file

### ISSUE THREAD

**Issue by nene** [TypeStrong/atom-typescript#1481]
I have a setup where the `tsconfig.json` file extends from `tsconfig.default.json`:

`tsconfig.json`:
```json
{
  "extends": "./tsconfig.defaults",
  "include": [
    "src/**/*.ts"
  ]
}
```

`tsconfig.defaults.json`:
```json
{
  "compileOnSave": true,
  "compilerOptions": {
    "module": "commonjs"
  }
}
```

When I move `compileOnSave` setting to the main `tsconfig.json` file, then atom typescript starts compiling on save. But when the setting is inside extended file, it has no effect.

**Comment by nene** [comment]
Hi @lierdakil,

unfortunately 13.0.0 hasn't fixed my problem. When I remove `compileOnSave` from `tsconfig.json`, atom-typescript stops compiling, even though the setting is defined in the config file from where it extends from.

**Comment by lierdakil** [comment]
Right, sorry. I had a misconception about some TypeScript internals apparently. This should be fixed in v13.0.1.

**Comment by nene** [comment]
This now works when I have:

```
{
  "extends": "./tsconfig.defaults.json"
}
```

But it doesn't work when there is no file extension specified:

```
{
  "extends": "./tsconfig.defaults"
}
```

The latter form is however supported by typescript compiler itself.

**Comment by lierdakil** [comment]
Thanks for pointing this out. Should be fixed in v13.0.2.

**Comment by nene** [comment]
Thanks a lot!

This now works perfectly for me.

Big thanks for solving this.
